### ISSUE TITLE
Internal lighting issue

### ISSUE THREAD

**Issue by Havner** [JPLogistics/JPLogistics_C152#154]
Don't know what is faulty and how to describe it so I'll just describe how to reproduce.

1. Load the plane at night, turn on battery
2. Turn on the white dome light -> see ![screenshot1](https://user-images.githubusercontent.com/6246347/137645298-6e6ff391-6103-4c37-a810-2a7f81beecfc.jpg)
3. Turn on panel lights (white PANEL LTS switch) -> see ![screenshot2](https://user-images.githubusercontent.com/6246347/137645296-f0798230-3cdc-46a5-901a-b59792c96891.jpg)

Turning on panel light made almost all the descriptions disappear.

**Comment by weptburrito** [comment]
Yup, known issue with a decal file and how the model interacts with it. Still trying to pin down the who what when and why.
